packages/components/src/components/vertical-tab/vertical-tab-header.tsx

```diff
@@ -67,10 +102,12 @@ const VerticalTabHeader = ({
             })}
             onClick={handleClick}
         >
-            <HeaderIcon icon={item.icon} is_active={is_active} />
-            <Header text={label} />
-            {children}
-            {item.component}
+            <>
```

Comment:
Just for my understanding why we need these fragments?

Reply:
<img width="921" alt="Screenshot 2023-02-20 at 10 53 04" src="https://user-images.githubusercontent.com/103182683/220045164-995722a1-e950-46f9-abb4-746801d23478.png">
it is weird , but I had this error

Reply:
Nice :smile:

Reply:
@yauheni-deriv let's remove this fragment but instead wrap only <>{item.component}</> into it then😊 it's gonna be less intrusive